afqsi_docker/docker-build.sh

```diff
@@ -0,0 +1,9 @@
+COMMIT=${1}
+COMMIT="$(echo "${COMMIT}" | tr -d '[:space:]')"
+export COMMIT
+TAG="ghcr.io/${2}/afqsi:${COMMIT}"
+TAG="$(echo "${TAG}" | tr -d '[:space:]')"
+echo $TAG
+docker build --no-cache -t $TAG --build-arg COMMIT ./afqsi_docker
+docker run --rm -i --entrypoint /usr/local/miniconda/bin/pytest $TAG /usr/local/miniconda/lib/python3.7/site-packages/AFQ/tests/test_api.py::test_AFQ_data_waypoint
```

Comment:
Does this mean that the test is run every time a user calls this script to build the docker?

Reply:
Or do we never use these scripts to build? The builds are only ever done on GitHub Actions?

Reply:
The latter. The user is not meant to use this script. We use this script to build. I want to test that pyAFQ installed correctly in the qsiprep environment everytime we make a new image. I could probably have found a way to write all of this inside the workflow file, but I am more familiar with bash syntax, so I wrote these two little shell scripts instead.